backend/types.py

```diff
@@ -157,7 +159,7 @@ class ParserConfig(ConfiguredBaseModel):
     Parser configuration
     """
 
-    name: str
+    name: str  # File extension
```

Comment:
What does this comment mean?

Reply:
the name here denotes the file extension when writing the parser config, this was just to identify that.